### PR TITLE
[WIP] bubble up postgres stats as cluster available stats

### PR DESCRIPTION
**Pull request by skahler-pivotal**
[WIP] Create gp_toolkit views that make some of the pg_stat counters available to be reviewed at a cluster level

TODO
~~still need to add indexes~~
add tests

### CODE REVIEW

**Comment by Eulerizeit** [comment]
Maybe dumb question. Can we create a function that writes SQL that generates all of these? 

i.e. if there is a pg_foo that is a Postgres view we can automagically create gp_view

**Comment by ivannovick** [comment]
I like the idea!  nice initiative

File: src/test/regress/expected/gp_toolkit.out [hunk 1]
```diff
@@ -37,7 +37,7 @@ from gp_toolkit.__gp_is_append_only iao
 inner join pg_catalog.pg_class c on iao.iaooid = c.oid
 where c.relname like 'toolkit_%'
 group by iaotype;
- count | iaotype
```
Comment (hlinnaka) [inline src/test/regress/expected/gp_toolkit.out]: Please leave out all these spurious whitespace changes. Whitespace is not significant right now, but we're just about merge commit ce3153fa93 from PostgreSQL 9.0, and with that, it becomes significant. (I'm just cleaning up a ton of whitespace differences in PR #4511, in preparation for that.)

Comment (skahler-pivotal) [reply src/test/regress/expected/gp_toolkit.out]: will do, I think the editor made those changes and they are actually breaking the test output checks. I'll go back and fix that

**Comment by skahler-pivotal** [comment]
going to close this for a cleaner version without whitespace changes

**Pull request closed without merging**